app/src/main/java/com/duckduckgo/app/location/ui/LocationPermissionsViewModel.kt

```diff
@@ -79,6 +82,12 @@ class LocationPermissionsViewModel(
         )
     }
 
+    fun loadFavicon(url: String, view: ImageView) {
+        viewModelScope.launch {
```

Comment:
similar concern as before around `viewModel` and `View` lifecycle.

Reply:
Fixed as explained above.